blinkpy/camera.py

```diff
@@ -98,14 +98,13 @@ def arm(self):
         """Return arm status of camera."""
         return self.motion_enabled
 
-    @arm.setter
-    def arm(self, value):
+    async def async_arm(self, value):
```

Comment:
Can't have an arm.setter in async.  Since the setting function has to be async, I couldn't keep the `arm.setter`.
It's a bit tricky some times- any lower level async will force async all the way up.